docs/authentication/jwt-auth.html

```diff
@@ -16,13 +16,407 @@
+        <pre><code class="language-swift">static let jwtSigner = JWTSigner.hs256(key: Data("kitura".utf8))</code></pre>
+        <h2 class="heading-2"><span class="blue-text">Step 2b:</span> Create your JWTVerifier</h2>
+        <p>The `JWTVerifier` is used to verify a signed JWT string.
+             This ensures the signature was generated using the same password and the JWT has not been changed.</p>
+        <p>Inside the app extension, we create an HMAC `JWTVerifier` by providing the same password as before:</p>
+        <pre><code class="language-swift">static let jwtVerifier = JWTVerifier.hs256(key: Data("kitura".utf8))</code></pre>
+      </div>
+      <div id="ecdsa" class="tabcontent">
+      <p class="block-text">You can process JWTs using Elliptic Curve Digital Signature Algorithm (ECDSA).
+            This uses asymmetric keys with a private key for signing and a public key for verifying.
+            The public key allows anyone to verify a JWT but only the provider can create them.
+            ECDSA is recommended over RSA since it uses smaller keys, requires less processing power and produces smaller signatures.</p>
+        <h2 class="heading-2"><span class="blue-text">Step 2a:</span> Create your elliptic curve keys</h2>
+        <p>Elliptic curve keys can be generated through
+             <a href="https://developer.apple.com/account/ios/authkey" rel="noopener" target="_blank">Apple</a>,
+             <a href="https://wiki.openssl.org/index.php/Command_Line_Elliptic_Curve_Operations#Generating_EC_Keys_and_Parameters" rel="noopener" target="_blank">OpenSSL</a>
+              or using <a href="https://github.com/IBM-Swift/BlueECC#elliptic-curve-private-key" rel="noopener" target="_blank">BlueECC</a>.
+             However for simplicity we will use some keys we generated earlier.</p>
+        <p>Add the following EC PEM Strings you your App extension:<p>
+    <pre><code class="language-swift">static let privateKey = """
+-----BEGIN EC PRIVATE KEY-----
+MHcCAQEEIJX+87WJ7Gh19sohyZnhxZeXYNOcuGv4Q+8MLge4UkaZoAoGCCqGSM49
+AwEHoUQDQgAEikc5m6C2xtDWeeAeT18WElO37zvFOz8p4kAlhvgIHN23XIClNESg
+KVmLgSSq2asqiwdrU5YHbcHFkgdABM1SPA==
+-----END EC PRIVATE KEY-----
+"""
+static let publicKey = """
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEikc5m6C2xtDWeeAeT18WElO37zvF
+Oz8p4kAlhvgIHN23XIClNESgKVmLgSSq2asqiwdrU5YHbcHFkgdABM1SPA==
+-----END PUBLIC KEY-----
+"""</code></pre>
+    <h2 class="heading-2"><span class="blue-text">Step 2b:</span> Create your JWTSigner</h2>
+    <p>The `JWTSigner` is used to sign a `JWT` instance and produce a signed JWT string.</p>
+    <p>Inside the App extension, create an ECDSA `JWTSigner` by providing the private key:</p>
+    <pre><code class="language-swift">static let jwtSigner = JWTSigner.es256(privateKey: Data(privateKey.utf8))</code></pre>
+    <div class="info">
+      <p>BlueECC is only available on OS X 10.13 or newer. You will need to handle this within your project.</p>
```

Comment:
"BlueECC is only available on OS X 10.13 or newer." Why are you telling me this? "You will need to handle this within your project." ??? It's not obvious whether we're using it here or not.
"macOS version 10.13" to me is clearer than OS X 10.13.... but that might just be me.